### DISCUSSION TITLE
actions/setup-node@v1: The process '/bin/tar' failed with exit code 2

### DISCUSSION THREAD

**Discussion by IvanBioRender** [community/community#25151] [Actions]
<p>Hello,</p>
<p>We’ve frequently been having the following test failure:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/8/8fddc83a1419ea3e2c31209baf71b5bd95a8cd9f.png" data-download-href="/uploads/short-url/kwHrxVpU3mx9vAOubpQUqFnlqxh.png?dl=1" title="setup-node-failed.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/55986695/181089163-0d209ff0-c3c3-43fd-8a79-7694825dc545.png" alt="setup-node-failed.png" data-base62-sha1="kwHrxVpU3mx9vAOubpQUqFnlqxh" width="690" height="200" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/8/8fddc83a1419ea3e2c31209baf71b5bd95a8cd9f_2_690x200.png, https://aws1.discourse-cdn.com/github/optimized/2X/8/8fddc83a1419ea3e2c31209baf71b5bd95a8cd9f_2_1035x300.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/8/8fddc83a1419ea3e2c31209baf71b5bd95a8cd9f_2_1380x400.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/8/8fddc83a1419ea3e2c31209baf71b5bd95a8cd9f_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">setup-node-failed.png</span><span class="informations">1467×426 71.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>There is an open issue for this (<a href="https://github.com/actions/setup-node/issues/116" rel="nofollow noopener">actions/setup-node#116</a> ), and it appears that other users are getting the same error, but there’s been no response from Github. I was hoping that posting it here would get some more visibility, an update at least saying “we’re looking into it” would be nice.</p>
<p>Thanks!</p>

**Comment by weide-zhou** [comment]
<p>Hi <a class="mention" href="/ivanbiorender">@ivanbiorender</a> ,</p>
<p>There is new comment on the <a href="https://github.com/actions/setup-node/issues/116" rel="nofollow noopener">issue</a>: It looks like error handling on dest stream (file stream), but not the HTTP response body stream. Working on it… Also looks like retry around getting a successful HTTP code, but no retry when reading body. </p>
<p>Now it’s being fixed, more details here: <a href="https://github.com/actions/toolkit/pull/369" rel="nofollow noopener">https://github.com/actions/toolkit/pull/369</a></p>
<p>Thanks a lot!</p>

**Comment by huan** [comment]
<p>I’m still expericing this isue today.</p>
<p>See: <a href="https://github.com/wechaty/wechaty/runs/494081908#step:3:16" rel="nofollow noopener">https://github.com/wechaty/wechaty/runs/494081908#step:3:16</a></p>

**Comment by IvanBioRender** [comment, marked as answer]
<p>Looks like the issue is fixed 🎉 <a href="https://github.com/actions/setup-node/issues/116" rel="nofollow noopener">https://github.com/actions/setup-node/issues/116</a></p>
